### PR TITLE
Improve some RESTful api details

### PR DESCRIPTION
**Pull request by Jerry098**
TODO: this pr need address comments later

### CODE REVIEW

File: clients/restful-api/edge.md [hunk 1]
```diff
@@ -14,6 +14,18 @@ EdgeId是由 `src-vertex-id + direction + label + sort-values + tgt-vertex-id` 
...
 
+Params说明
+
+- label：边类型名称，必填
+- outV：源顶点id，必填
+- inV：目标顶点id，必填
+- outVLabel：源顶点类型。必填
+- inVLabel：目标顶点类型。必填
+- properties: 边关联的属性，对象内部结构为：
+- name：属性名称
```
Comment (javeme) [inline clients/restful-api/edge.md:25]: 增加缩进

File: clients/restful-api/gremlin.md [hunk 2]
```diff
@@ -5,7 +5,7 @@
 ##### Params
 
 - gremlin: 要发送给`HugeGraphServer`执行的`gremlin`语句
-- bindings: 可以给`gremlin`语句中的变量绑定值
+- bindings: 用来绑定参数，key是字符串，value是绑定的值（只能是字符串或者数字），功能类似于MySQL的preparedstatement，即预编译和缓存语句，加速运行
```
Comment (javeme) [inline clients/restful-api/gremlin.md:8]: Prepared Statement
用于加速语句执行

File: clients/restful-api/vertex.md [hunk 3]
```diff
@@ -1,13 +1,14 @@
 ### 2.1 Vertex
 
-0.4 版本对顶点 Id 做了比较大的改动，VertexLabel 中的 Id 策略决定了顶点的 Id 类型，其对应关系如下：
+顶点类型中的 Id 策略决定了顶点的 Id类型，其对应关系如下：
```
Comment (javeme) [inline clients/restful-api/vertex.md:3]: 保持Id后的空格

**Review by javeme — APPROVED** [review]

**Review by imbajin — APPROVED** [review]
merge now & fix the comments later

**Pull request merged**